SocialStudies/SurviveTheMiddleAges/survive-the-middle-ages.ipynb

```diff
@@ -0,0 +1,377 @@
+    "marry_not = random.choice(marriage_probablity)\n",
+    "print('This year my character will', marry_not)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Birth\n",
+    "\n",
+    "If your character is married and at least 12 years old, run the following cell to determine if they have a child or not (each with a probability of 50%), and the gender of their child.\n",
+    "\n",
+    "If a baby is born and survives (50% probability), give them a name and add them to your character's family with a starting age of 0."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "birth = random.choice(['yes', 'no'])\n",
+    "if birth == 'yes':\n",
+    "    child_gender = random.choice(['boy', 'girl'])\n",
+    "    print('My character gives birth to a baby', child_gender)\n",
+    "    survive = random.choice(['yes', 'no'])\n",
+    "    if survive == 'no':\n",
+    "        print('but unfortunately the baby does not survive.')\n",
+    "else:\n",
+    "    print('My character does not give birth this year.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plague\n",
+    "\n",
+    "The [Black Death](https://en.wikipedia.org/wiki/Black_Death) killed many people in Europe between [1347 and 1351](https://www.history.com/news/black-death-timeline).\n",
+    "\n",
+    "![plague map](https://upload.wikimedia.org/wikipedia/commons/4/4e/1346-1353_spread_of_the_Black_Death_in_Europe_map.svg)\n",
+    "*Image from [Wikipedia - Flappiefh](https://commons.wikimedia.org/wiki/File:1346-1353_spread_of_the_Black_Death_in_Europe_map.svg)*\n",
+    "\n",
+    "In this game your main character was born in 1335, so the plague come to Europe when they are 12 years old.\n",
+    "\n",
+    "Set the `year` and `number_of_family_members` in the following code cell, and run the cell to determine the outcomes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "year = 1347\n",
+    "number_of_family_members = 4\n",
+    "\n",
+    "def infection():\n",
+    "    infected_countries = []\n",
+    "    plague_outcomes = ['survives']\n",
+    "    if year == 1347:\n",
+    "        infected_countries = ['Italy', 'The Balkans[b]']\n",
+    "    if year == 1348:\n",
+    "        infected_countries = ['Italy','The Balkans[b]','Germany','France','England and Wales','Spain and Portugal','Austria-Hungary[a]','Switzerland','Belgium','North-Eastern Europe[c]']\n",
+    "    if year == 1350 or year == 1351:\n",
+    "        infected_countries = population['Country/Region'].to_list()\n",
+    "    if country in infected_countries:\n",
+    "        plague_outcomes = ['dies', 'survives']\n",
+    "    plague_outcome = random.choice(plague_outcomes)\n",
+    "    return plague_outcome\n",
```

Comment:
this definitely seems like code that could be much better appreciated, maybe add some more noisy output? It's very cool

Reply:
Thank you for all the suggestions and comments, very helpful. Not sure what you mean by noisy output here though.

Reply:
Oh, perhaps just something like 
` During this time xyz countries were infected`

Reply:
Great idea